src/app/core/store/authors/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as AuthorsActions from "./authors.actions";
+
+export * from "./authors.state";
+export * from "./authors.reducer";
+export * from "./authors.effects";
+export * from "./authors.facade";
+export * from "./authors.selectors";
```

Comment:
selector should be exported the same way as actions
If your application will become lib - there will be possibility to use selectors and actions without facade modification.
`import * as AuthorsSelectors from "./authors.selectors;`